float32 roll
float32 pitch
float32 yaw
float32 duration
---
bool status
